documentation/docs/migration-guide.md

```diff
@@ -74,20 +77,67 @@ Access to node information such as name, bounding box and properties/attributes
 
 The API for accessing node information has not changed `@cognite/3d-viewer`, but since Reveal now uses tree indices explicit mapping tree index to node ID is necessary.
 
-## Working with asset mappings
+### Working with asset mappings
 
 [3D Asset Mappings](https://docs.cognite.com/api/v1/#tag/3D-Asset-Mapping) enables mapping 3D nodes to [assets](https://docs.cognite.com/api/v1/#tag/Assets). A mapping is a link from `assetId` to `nodeId`, so to use these with Reveal it's necessary to [map these to tree indices](#accessing-node-information-and-mapping-between-node-ids-and-tree-indices).
 
-## Ray picking and intersection for handling click events
+### Ray picking and intersection for handling click events
 
 In `@cognite/3d-viewer` `Cognite3dViewer.getIntersectionFromPixel` optionally accepts a `model`-argument to restrict the result to a single model. Support for this has been removed 
 in `@cognite/reveal`. Previously `getIntersectionFromPixel` would return a struct with both `nodeId` and `treeIndex`. Now this has been changed to only include `treeIndex` (to 
 determine `nodeId` use `async Cognite3DModel.mapNodeId(nodeId)`).
 
-## Other differences
+### Other differences
 
 There are a few other noticeable changes from `@cognite/3d-viewer` and `@cognite/reveal`:
 
 - `@cognite/3d-viewer` supports local caching to reduce the time to load previously opened 3D models. Currently, this is not supported by `@cognite/reveal`, but the need for such functionality is reduced by adding streaming capabilities.
 - In `@cognite/3d-viewer` `Cognite3DViewer.addModel(...)` will always return a `Cognite3DModel`. In `@cognite/reveal` this function might also return a `CognitePointCloudModel`. To explicitly add a CAD model or point cloud model use `Cognite3DViewer.addCadModel(...)` or `Cognite3DViewer.addPointCloudModel(...)`
-- `Cognite3DViewer.loadCameraFromModel(...)`] has been added for loading camera settings from CDF when available.
\ No newline at end of file
+- `Cognite3DViewer.loadCameraFromModel(...)`] has been added for loading camera settings from CDF when available.
+
+## Preparing models
+
+`@cognite/reveal` requires new output file formats which means that 3D models that have been converted before June 2020 might need reconversion. To determine if a model needs reconversion, the [listModelRevisionOutputs API endpoint](...) can be used with URL parameter `format=reveal-directory`. This can be done using the SDK, e.g.:
+```jsx
+// Switch these with your project/model identifier
+const project = 'publicdata';
+const modelId = 4715379429968321;
+const revisionId = 5688854005909501; 
+
+// Prepare a SDK client and authenticate
+const sdk = new CogniteClient({ appId: 'cognite.reveal.docs.IsModelCompatible' });
+sdk.loginWithOAuth({ project });
+await sdk.authenticate();
+
+// Request a list of formats
+const url = `https://api.cognitedata.com/api/v1/projects/${project}/3d/models/${modelId}/revisions/${revisionId}/outputs`;
```

Comment:
Not related to that PR but still:

Shouldn't we have methods in the sdk for that? Like 

```
sdk.3d.getModel(modelId, revisionId, { params: { format: 'reveal-directory' } })
```

This magical check `response.data.items.length > 0` also seems a bit odd to me. Feels more like something that should be hidden behind a nicer interface. 

If not sdk, maybe we should have some method in the Reveal? Some public utility function? But I think sdk is a better place for that.

Reply:
I agree with this being a bit awkward. Should probably have a `hasModelOutput` function in SDK - but wont fix that now.

Reply:
@larsmoa Here is the comment referred to